pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "Flask-Phrase"
+version = "2.0.0"
+authors = [
+  { name="Phrase", email="info@phrase.com" },
```

Comment:
indentation here looks like 2 spaces instead of tab (4 spaces) like some of the other indentations